52/ttafsir/pomodoro.py

```diff
@@ -0,0 +1,39 @@
+from rich.console import Console
```

Comment:
`rich`?

Reply:
I've been playing with `rich` lately and it seems super cool and useful for CLI outputs and layouts.

Reply:
cool, need to check that out, maybe we can do an article on the blog ...